uk/medium-promise-all.md

```diff
@@ -60,7 +60,7 @@ declare function PromiseAll<T extends unknown[]>(values: readonly [...T]): Promi
 declare function PromiseAll<T extends unknown[]>(values: readonly [...T]): Promise<T extends Promise<infer R> ? R : T>
 ```
 
-Рішення досі неробоче, тому що `T`, не об'єднання, а кортеж.
+Рішення досі помилкове, тому що `T`, не об'єднання, а кортеж.
```

Comment:
What is wrong with the dot? 🤔 

```suggestion
Рішення досі помилкове, тому що `T`, не об'єднання, а кортеж.
```

Reply:
![image](https://user-images.githubusercontent.com/23432891/123508086-e1a2e980-d675-11eb-85f3-c3f81e3cd288.png)

Reply:
Strange, really strange

Reply:
Am I the only with the highlighted dot in red?